atlasreader/atlasreader.py

```diff
@@ -265,6 +272,37 @@ def get_subpeak_coords(clust_img, min_distance=20):
     return coords
 
 
+def check_atlas_bounding_box(voxIDs, box_shape):
+    """
+    Returns the provided voxel ID if the voxel is inside the bounding box of
+    the atlas image, otherwise the voxel ID will be replaced with the origin.
+
+    Parameters
+    ----------
+    voxIDs : (N, 3) numpy.ndarray
+        `coords` in cartesian space
+    box_shape : (3,) list of int
+        size of the atlas bounding box
+    prob_thresh : [0, 100] int, optional
```

Comment:
It seems like this parameter isn't used so I think it can be removed from the doc-string!